pkg/operator/ceph/pool/radosnamespace/controller.go

```diff
@@ -366,6 +371,10 @@ func (r *ReconcileCephBlockPoolRadosNamespace) createOrUpdateRadosNamespace(ceph
 	namespacedName := fmt.Sprintf("%s/%s", cephBlockPoolRadosNamespace.Namespace, cephBlockPoolRadosNamespace.Name)
 	logger.Infof("creating ceph blockpool rados namespace %q", namespacedName)
 
+	if getRadosNamespaceName(cephBlockPoolRadosNamespace) == "" {
+		logger.Info("can't create empty radosnamepace as it is already present.")
```

Comment:
Log messages should not end in periods.

Log messages should include enough info about the current underlying resource and/or the kube object namespace and name to make sure it is clear when debugging which resource is being referenced. Please add enough info to the info output to ensure the log can be correlated to the right inputs and outputs without guessing. 

I would suggest including both the CephBlockPoolRadosNamespace <namespace>/<name> and underlying Ceph namespace to ensure debugging can be done quickly without too much lookup.